config/index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
```

Comment:
Please check license dates (2020 :smile: )

Reply:
And careful, it's `PrestaShop SA and Contributors`

Reply:
fixed ;)